src/main/java/core/basesyntax/serviceimpl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.storage.Storage;
+
+public class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
The class `ReportGeneratorImpl` should be placed in a package that reflects its purpose, such as `service.impl` according to the project structure guidelines.